types/is-empty/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for is-empty 1.2
+// Project: https://github.com/ianstormtaylor/is-empty
+// Definitions by: Stanislav Termosa <https://github.com/termosa>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Test if a value is empty
+ */
+declare function isEmpty(value: any): boolean;
+
+export default isEmpty;
```

Comment:
`export = isEmpty` is more accurate based on the implementation at https://github.com/ianstormtaylor/is-empty/blob/master/lib/index.js

Corresponding import statement would be `import isEmpty = require('is-empty')`